Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/ScreenSpace/ScreenSpaceUtil.azsli

```diff
@@ -45,3 +45,62 @@ float4 WorldPositionFromDepthBufferMS(const Texture2DMS<float2> depthStencilText
     depthStencilTexture.GetDimensions(dimensions.x, dimensions.y, sampleCount);
     return WorldPositionFromDepthBuffer(dimensions, zDepth, screenCoords);
 }
+
+//! Calculates the view position and normal from the fragment position.
+void ViewPositionAndNormalFromDepth(const Texture2DMS<float2> depthStencilTexture, const VSOutput IN, out float3 outPositionWS, out float3 outNormalWS)
```

Comment:
Thanks for the change, but I'm not seeing a corresponding change for the fullscreen shadow code to actually use this. (reminder, shader code needs the exact same discipline as normal C++ code and test code).